BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
+license:
+- url # REQUIRED
```

Comment:
Potential controlled source? https://spdx.org/licenses/

Reply:
Ooh, nice source of license controlled vocabulary. 
1. Should we log an identifer for the license here, AND the url?
2. Custom licenses: Right now in BICCN the policy is the data should all be CC-BY-4.0 if generated by BICCN. HOWEVER, its possible at some point that we ingest some data that has a different license attached for some reason. If that license is a custom license and not included in that list, what do we use this url for?